src/redis_manager.py

```
@@ -121,20 +135,53 @@ def pop(self, timeout=5):
+        return None, None, None
+
+    def ack(self, payload):
+        if not payload:
+            return False
+        if not self.client:
+            self.connect()
+            if not self.client:
+                return False
+        try:
+            self.client.lrem(self.processing_key, 1, payload)
+            return True
```

Ruff (0.15.15)

[warning] Consider moving this statement to an `else` block

(TRY300)